packages/angular/cli/commands/deploy-long.md

```diff
@@ -1,9 +1,22 @@
-The `ng deploy` command is a shortcut for:
+The command takes an optional project name, as specified in the `projects` section of the `angular.json` workspace configuration file.
+When a project name is not supplied, executes the `deploy` builder for the default project.
 
-```
-ng run [PROJECT_NAME]:deploy
-```
+To use the `ng deploy` command, use `ng add` to add a package that implements deployment capabilities to your favorite platform.
+Adding the package automatically updates your workspace configuration, adding a deployment
+`CLI builder`(guide/cli-builder).
```

Comment:
Users see `deploy-long.md` when the run `ng deploy --help` as well and this won't be rendered properly. Maybe we can drop it?

```suggestion
`CLI builder`.
```

Reply:
OK -- but I think the long description only shows up in AIO, not in the help.

Reply:
Yep, feel free to ignore my comment.